README.md

```diff
@@ -11,3 +11,38 @@ La idea del proyecto viene de querer predecir el dinero que generarán los usuar
 ## Documentación :bookmark_tabs:
 
 Para acceder a toda la documentación puede acceder desde [aquí](docs/)
+
+## Task runner para el proyecto
+
+Como este proyecto se esta implementando el Go, se ha buscado un task runner para este. El elegido es [Task](https://taskfile.dev/#/). Para ver más información sobre el gestor de tareas y dependecias pinche [aquí](/docs/gestor.md).
+
+1. Necesitamos tener instalado tener instalado en nuestro sistema **Go**. Si no esta instalado se puede hacer desde [aquí](https://golang.org/doc/install).
+
+2. Se tiene que instalar el gestor de tareas, hay varias maneras de [instalarlo](https://taskfile.dev/#/installation) aqui se va a comentar una manera de todas las posibles. La que me ha resultado más sencilla, obteniendo el binario de la herramienta, es la siguiente:
```

Comment:
Esto está copiado de la página de instalación. Pero, con eso, ¿qué privilegios se necesitan para instalarlo? ¿Hay otra alternativa?

Reply:
> Aparte de los comentarios, no hay ningún issue que plantee ningún problema, ni se desarrolla contra él. No se sabe si el código desarrolla la lógica de negocio o qué.

He creado issues y un milestone donde comento la necesidad de elegir un task runner y los problemas que han ido surgiendo durante el objetivo.

Reply:
¿Y por qué no está este PR en ese milestone? ¿Podéis intentar, por favor, entender por qué se hacen las cosas y no simplemente hacerlas?